egs/mgb2_arabic/s5/local/process_xml.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
I didn't check this file but please include the changes from the previous py file -- I assume the files are either identical or very similar